applications/crossbar/src/crossbar_doc.erl

```diff
@@ -284,15 +284,15 @@ load_view(View, Options, Context, StartKey, PageSize) ->
     load_view(View, Options, Context, StartKey, PageSize, 'undefined').
 
 load_view(View, Options, Context, StartKey, PageSize, FilterFun) ->
+    Dbs = props:get_value('databases', Options, [cb_context:account_db(Context)]),
+    ExistingDbs = lists:filter(fun couch_mgr:db_exists/1, Dbs),
```

Comment:
@jamesaimonetti 
I removed filter and got error.
Also this code still exists in master branch. This modification is only for better looking. I can revert it :)